src/reactviews/pages/ExecutionPlan/findNodes.tsx

```diff
@@ -0,0 +1,128 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { useContext, useState } from 'react';

```

Comment:
@laurennat , have you tried using something like popover from fluent control?
https://react.fluentui.dev/?path=/docs/components-popover--docs